packages/manager/modules/exchange/src/external-contact/external-contact.html

```diff
@@ -3,220 +3,117 @@
     data-ng-controller="ExchangeTabExternalContactsCtrl as ctrl"
 >
     <div class="row">
-        <div class="col-md-9">
-            <form
-                class="form-inline d-md-flex justify-content-md-end mb-3"
-                name="searchExternalContactForm"
+        <div class="col-md-12">
+            <oui-datagrid
+                rows-loader="ctrl.loadContacts($config)"
+                page-size="10"
             >
-                <div class="form-group">
-                    <label
-                        class="sr-only"
-                        for="externalContactSearch"
-                        data-translate="common_search"
-                    ></label>
-                    <div class="input-group">
-                        <input
-                            type="text"
-                            class="form-control"
-                            id="externalContactSearch"
-                            maxlength="256"
-                            name="externalContactSearch"
-                            placeholder="{{::'exchange_tab_EXTERNAL_CONTACTS_search_placeholder' | translate}}"
-                            data-ng-change="ctrl.onSearchValueChange()"
-                            data-ng-disabled="ctrl.contactsLoading"
-                            data-ng-model="ctrl.filter"
-                            data-ng-model-options="{ debounce: 800 }"
-                        />
-                        <div class="input-group-btn" data-ng-if="ctrl.filter">
-                            <button
-                                class="btn btn-default"
-                                type="button"
-                                aria-label="{{'common_cancel' | translate}}"
-                                data-ng-click="ctrl.filter = '';ctrl.onSearchValueChange()"
-                                data-ng-disabled="ctrl.contactsLoading"
-                            >
-                                <span
-                                    class="fa fa-times"
-                                    aria-hidden="true"
-                                ></span>
-                            </button>
-                        </div>
+                <oui-column
+                    title="'exchange_tab_EXTERNAL_CONTACTS_table_headers_externalEmailAddress' | translate"
```

Comment:
```suggestion
                    data-title="'exchange_tab_EXTERNAL_CONTACTS_table_headers_externalEmailAddress' | translate"
```

`title` is a `html5` attribute 🙂